src/browser/settings.jsx

```diff
@@ -56,6 +71,14 @@ var SettingsPage = React.createClass({
       var currentWindow = remote.getCurrentWindow();
       currentWindow.setAutoHideMenuBar(config.hideMenuBar);
       currentWindow.setMenuBarVisibility(!config.hideMenuBar);
+
+      var autostart = this.state.autostart;
+      appLauncher.isEnabled().then(function(enabled) {
```

Comment:
Just curious, is this confirmation necessary? I think that simple enable/disable are enough in this part.